frontend/src/components/pages/FridgeManagement/FridgeCheckIns/index.tsx

```diff
@@ -83,7 +91,7 @@ const CreateCheckIn = () => {
     };
 
     const res = await CheckInAPIClient.createCheckIn(checkInData);
-    if (!res) {
+    if (!res.id) {
```

Comment:
why not return as false on failure here as well to stay consistent with `updateCheckIn`?